jplag/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -84,6 +84,11 @@ private static List<ComparisonReport> generateComparisonReports(JPlagResult resu
         return comparisons;
     }
 
+    private static List<JPlagComparison> getComparisons(JPlagResult result) {
+        int maxNumOfComparisons = result.getOptions().getMaximumNumberOfComparisons();
```

Comment:
I prefer a short name without abbreviations like `numberOfComparisons` to `maxNumOfComparisons`.